results-processor/main.py

```diff
@@ -115,27 +141,21 @@ def task_handler():
     params = flask.request.form
     # Mandatory fields:
     uploader = params['uploader']
-    gcs_path = params['gcs']
+    gcs_paths = params.getlist('gcs')
     result_type = params['type']
     # Optional fields:
     labels = params.get('labels', '')
 
-    # TODO(Hexcles): Support multiple results.
-    assert result_type == 'single'
-
-    match = re.match(r'/([^/]+)/(.*)', gcs_path)
-    assert match
-    bucket_name, blob_path = match.groups()
-
-    gcs = storage.Client()
-    bucket = gcs.get_bucket(bucket_name)
-    blob = bucket.blob(blob_path)
+    assert (
+        (result_type == 'single' and len(gcs_paths) == 1) or
+        (result_type == 'multiple' and len(gcs_paths) > 1)
+    )
 
-    with tempfile.NamedTemporaryFile(suffix='.json') as temp:
-        blob.download_to_file(temp)
-        temp.seek(0)
-        report = wptreport.WPTReport()
-        report.load_json(temp)
+    report = wptreport.WPTReport()
+    for gcs_path in gcs_paths:
+        if not _process_chunk(report, gcs_path):
+            # The chunk is invalid; return 2XX to prevent retry.
```

Comment:
Will we have a log of this incident later, or how will we notice if the processor starts rejecting some input?